src/components/Product/AbTesting/index.tsx

```diff
@@ -304,13 +307,18 @@ export const ProductAbTesting = () => {
                     />
                 </div>
 
-                <section id="customers" className="-mt-20">
-                    <ul className="list-none p-0 grid md:grid-cols-3 gap-4 mb-10 md:mb-20">
+                <section id="customers" className="-mt-36 pt-36">
```

Comment:
Intentionally adding more padding here? I think this is fine to keep as it was

Reply:
I just copied this directly from other product pages that have four customer stories on them... I don't know what it does. 😅

Given this is how it is on other pages, I'm going to assume it's correct!